### PR TITLE
[full-screen] feat: allow expand in innerce images

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: e2e/tests/media.e2e.js [hunk 1]
```diff
@@ -122,24 +123,42 @@ describe('plugins', () => {
...
+        cy.loadOutOfViewImagesInGallery();
+        cy.waitForGalleryImagesToLoad();
+        cy.eyesCheckWindow({ tag: this.test.title, target: 'window', fully: false });
+        cy.get('[data-hook=fullscreen-root] [data-hook=image-item]', {
+          timeout: 10000,
+        }).should('be.visible');
+        cy.get(`[data-hook=${'nav-arrow-back'}]`)
+          .click({ force: true })
+          .wait(200)
+          .click();
```
Comment (shaulgo) [inline e2e/tests/media.e2e.js:144]: 1. what does the second click do?
2. The test clicks on the nav-arrow but doesn't check that anything happened?

File: e2e/tests/media.e2e.js [hunk 2]
```diff
@@ -122,24 +123,42 @@ describe('plugins', () => {
...
-            timeout: 10000,
-          }
-        ).click({ force: true });
-        cy.get(
-          '#pgiea8ec1609e052b7f196935318316299d_1 > :nth-child(1) > .gallery-item-wrapper > :nth-child(1) > a > .gallery-item-content > .gallery-item-visible',
-          {
-            timeout: 10000,
-          }
-        ).should('be.visible');
+        cy.get('[data-hook=ricos-viewer] [data-hook=item-wrapper]', {
```
Comment (shaulgo) [inline e2e/tests/media.e2e.js:154]: Please fix other places with the specific selectors. Like this one #pgiea8ec1609e052b7f196935318316299d_1

**Review by shaulgo — CHANGES_REQUESTED** [review]
> Comparison bundleSizes:
> Packages that shrank:
> wix-rich-content-plugin-accordion: 167KB => 166KB
> wix-rich-content-plugin-code-block: 175KB => 174KB
> wix-rich-content-plugin-gallery: 436KB => 435KB
> wix-rich-content-plugin-giphy: 165KB => 164KB
> wix-rich-content-plugin-headings: 159KB => 158KB
> wix-rich-content-plugin-line-spacing: 160KB => 159KB
> wix-rich-content-plugin-link-preview: 176KB => 175KB
> wix-rich-content-plugin-link: 163KB => 162KB
> wix-rich-content-plugin-social-polls: 316KB => 315KB
> wix-rich-content-plugin-sound-cloud: 247KB => 246KB
> wix-rich-content-plugin-table: 235KB => 234KB
> 
> Please update the baseline files by running locally "npm run saveBundlesSizesBaseline" and push the changes.

Please update the bundleSizes

File: e2e/tests/media.e2e.js (hunk 1 above)
Comment (tamrat-bay) [reply e2e/tests/media.e2e.js]: fixed

File: e2e/tests/media.e2e.js (hunk 2 above)
Comment (tamrat-bay) [reply e2e/tests/media.e2e.js:155]: done

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: e2e/tests/media.e2e.js [hunk 3]
```diff
@@ -122,36 +123,56 @@ describe('plugins', () => {
...
+          timeout: 10000,
+        })
+          .eq(1)
+          .click({ force: true });
+        cy.get('[data-hook=fullscreen-root] [data-hook=image-item]', {
+          timeout: 10000,
+        }).should('be.visible');
         // cy.eyesCheckWindow({
```
Comment (tamrat-bay) [inline e2e/tests/media.e2e.js:167]: check if we can uncomment these eye checks

**Pull request merged**